dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -95,11 +166,7 @@ window.Dashboard = {
 	__handleAppEvent: function (event) {
 		switch (event.name) {
 			case "CONFIG_READY":
-				if (window.location.protocol === "http:" && this.config.INSTALL_CERT) {
-					window.location.href = window.location.href.replace("http", "https");
```

Comment:
As far as I can tell this problem does not originate from the new logic (the events don't get fired) but rather from removing this line. I'm not 100% sure though.